borg/archiver.py

```diff
@@ -652,9 +648,10 @@ def preprocess_args(self, args):
 
     def build_parser(self, args=None, prog=None):
         common_parser = argparse.ArgumentParser(add_help=False, prog=prog)
-        common_parser.add_argument('-v', '--verbose', dest='verbose', action='store_true', default=False,
-                                   help='verbose output')
-        common_parser.add_argument('--log-level', dest='log_level', default='info', metavar='LEVEL',
+        common_parser.add_argument('-v', '--verbose', dest='log_level',
+                                   action='store_const', const='info', default='warning',
+                                   help='verbose output, same as --log-level=info')
+        common_parser.add_argument('--log-level', dest='log_level', default='warning', metavar='LEVEL',
```

Comment:
with "checking" i didn't just mean running the tests, but really using it interactively.